client/src/game/ui/menu/menu.vue

```diff
@@ -324,14 +358,18 @@ DIRECTORY.CSS changes
 .menu-accordion-subpanel > * {
     padding: 5px;
     display: flex;
-    justify-content: space-evenly;
     align-items: center;
+    text-align: left;
```

Comment:
I would apply this change only on the markers part for now.

Some redesign of the side bar is needed in the future, but having all the other elements also left-aligned is kinda weird imo.

(same thing applies to the removal of the justify-content)

Reply:
Done.